### DISCUSSION TITLE
Virus Warning because of flow launcher blocks it from running

### DISCUSSION THREAD

**Discussion by ulspot** [Flow-Launcher/Flow.Launcher#999] [General]
Hi, 
today my anti virus program (ESET Endpoint) warned me and blocked flow launcher from running. 

This is the Scan Log:

Version of detection engine: 24687 (20220127)
Date: 27.01.2022  Time: 10:22:37
Scanned disks, folders and files: C:\Users\[JonDoe]\Desktop\Flow-Launcher-Portable\FlowLauncher\app-1.9.1;C:\Users\[JonDoe]\Desktop\Flow-Launcher-Portable\FlowLauncher\packages;C:\Users\[JonDoe]\Desktop\Flow-Launcher-Portable\FlowLauncher\app.ico;C:\Users\[JonDoe]\Desktop\Flow-Launcher-Portable\FlowLauncher\Flow.Launcher.exe;C:\Users\[JonDoe]\Desktop\Flow-Launcher-Portable\FlowLauncher\Update.exe
C:\Users\[JonDoe]\Desktop\Flow-Launcher-Portable\FlowLauncher\packages\FlowLauncher-1.9.1-full.nupkg » ZIP » lib/net5.0-windows/Flow.Launcher.exe - Win32/AtlasVPN.A potentially unwanted application - cleaned by deleting [1]
Number of scanned objects: 2153
Number of detections: 1
Number of cleaned objects: 1
Time of completion: 10:23:20  Total scanning time: 43 sec (00:00:43)

Notes:
[1] Object has been deleted as it only contained the virus body.

Do you know anything about this or can you help? 

btw It is really a great tool and time saver - Many thanks for the great work and effort you guys put into this.

Best regards,
Ulspot

**Comment by deefrawley** [comment]
Hi Ulspot,

Could you please submit this to ESET as a false positive by following ESET's instructions here:

https://support.eset.com/en/kb141-submit-a-virus-website-or-potential-false-positive-sample-to-the-eset-lab?ref=esf

Thanks very much,
D

**Reply by ulspot** [reply]
This was the first thing I’ve done. No response until now.

**Reply by ulspot** [reply]
Ok. It seems to be resolved by ESET. Everything is working fine again.

**Reply by deefrawley** [reply]
Brilliant. Thanks for your help Ulspot and if you have any suggestions keep them coming here in the discussions or on our active Discord
